docs/admin-manual/setup/upgrade.md

```diff
@@ -39,10 +45,22 @@ You might need to run steps before and after the install procedure. Be sure to f
 
 :::
 
+:::caution
+
+If you are migrating LibreTime to a new server, you should read the upgrade procedure but must not apply them.
```

Comment:
May add something about when to apply them?

Reply:
This is already defined few lines above in the "normal" upgrade procedure (when we don't migrate to a new server):

![image](https://user-images.githubusercontent.com/19195485/225751571-33a960b5-c101-4fc3-bd48-e346693b5dc2.png)